### ISSUE TITLE
Conflicting required go versions

### ISSUE THREAD

**Issue by magickaiyang** [vusec/kasper#8]
Hello,

I'm a PhD student in computer science at Carnegie Mellon University. I'm trying to run Kasper by following your readme. I installed go 1.15.15 as per readme, but it seems that syzkaller now requires newer go versions. I tried newer go versions, but other errors were thrown. The error log is attached. Please advise. 

Best,
Kaiyang Zhao

```
kaiyang2@entropy /d/k/kasper ❯❯❯ go version
go version go1.15.15 linux/amd64
kaiyang2@entropy /d/k/kasper ❯❯❯ task syzkaller:build
task: [syzkaller:build] go get github.com/cbroglie/mustache/...
go: downloading github.com/cbroglie/mustache v1.4.0
go: found github.com/cbroglie/mustache/... in github.com/cbroglie/mustache v1.4.0
go: downloading github.com/spf13/cobra v1.7.0
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/inconshreveable/mousetrap v1.1.0
task: [syzkaller:build] echo "KERNEL: $KERNEL
SYZKALLER_PREFIX: $SYZKALLER_PREFIX
SYZKALLER_WORKDIR: /data/kaiyang/kasper/out/syzkaller-workdir
SYZKALLER_IMG: $SYZKALLER_IMG
SYZKALLER_LOGS_PATH: $SYZKALLER_LOGS_PATH" | $GOPATH/bin/mustache $ROOT/syzkaller.cfg.template > $SYZKALLER_CFG
task: [syzkaller:build] mkdir -p $ROOT/out/go/src/github.com/google
task: [syzkaller:build] ln -sfn $SYZKALLER_PREFIX $ROOT/out/go/src/github.com/google/syzkaller
task: [syzkaller:build] cd $SYZKALLER_PREFIX && make
go: downloading github.com/google/syzkaller v0.0.0-20230726144346-41fe1bae463b
go: downloading golang.org/x/sys v0.10.0
# github.com/google/syzkaller/pkg/image
../out/go/pkg/mod/github.com/google/syzkaller@v0.0.0-20230726144346-41fe1bae463b/pkg/image/compression.go:39:26: undefined: io.Discard
../out/go/pkg/mod/github.com/google/syzkaller@v0.0.0-20230726144346-41fe1bae463b/pkg/image/compression.go:60:18: undefined: io.ReadAll
note: module requires Go 1.19
# golang.org/x/sys/unix
../out/go/pkg/mod/golang.org/x/sys@v0.10.0/unix/mremap.go:33:10: undefined: unsafe.Slice
../out/go/pkg/mod/golang.org/x/sys@v0.10.0/unix/syscall.go:83:16: undefined: unsafe.Slice
../out/go/pkg/mod/golang.org/x/sys@v0.10.0/unix/syscall_linux.go:1018:20: undefined: unsafe.Slice
../out/go/pkg/mod/golang.org/x/sys@v0.10.0/unix/syscall_linux.go:2316:9: undefined: unsafe.Slice
../out/go/pkg/mod/golang.org/x/sys@v0.10.0/unix/syscall_unix.go:118:7: undefined: unsafe.Slice
../out/go/pkg/mod/golang.org/x/sys@v0.10.0/unix/sysvshm_unix.go:33:7: undefined: unsafe.Slice
note: module requires Go 1.17
Makefile:35: *** syz-env failed.  Stop.
task: Failed to run task "syzkaller:build": exit status 2
```

**Comment by magickaiyang** [comment]
My fix is in #9 .

**Comment by Jakob-Koschel** [comment]
Thanks for making the PR!
